### ISSUE TITLE
Changing db port - Can't Connect 

### ISSUE THREAD

**Issue by IAPOLINARIO** [SonarSource/docker-sonarqube#31]
When I map the default port (5432) of Postgres to any other port (5433, for example) the SonarQube container can't connect to the database and displays the following error:  

> java.lang.IllegalStateException: Can not connect to database. Please check connectivity and settings (see the properties prefixed by 'sonar.jdbc.').

My docker compose is simple just like the example below:

```
sonarqube:
   image: sonarqube
   ports:
   - "19000: 9000"
   - "19092: 9092"
   links:
     - sonardb
   environment:
      - SONARQUBE_JDBC_URL=jdbc:postgresql://sonardb:5433/sonar

sonardb:
   image: postgres:latest
   ports:
     - "5433: 5432"
   environment:
     - POSTGRES_USER=sonar
     - POSTGRES_PASSWORD=sonar
```
All the other configuration is set to default. If I just change the port mapping for "5432: 5432", the Sonar container connect in Postgres container without problems.

**Comment by Godin** [comment]
@IAPOLINARIO 
when containers linked they can communicate internally without expose of any ports - see "Communication across links" at https://docs.docker.com/engine/userguide/networking/default_network/dockerlinks/
and
```
   ports:
     - "5433:5432"
```
affects how port will published and not how it will be visible for linked container.

So that using
```
$ docker images
REPOSITORY          TAG                       IMAGE ID            CREATED             SIZE
sonarqube           latest                    334aa4aa378e        8 days ago          780.4 MB
postgres            latest                    247a11721cbd        2 weeks ago         265.9 MB

$ docker --version        
Docker version 1.11.0, build 4dc5990

$ docker-compose --version
docker-compose version: 1.5.1
```
with `docker-compose.yml`
```
sonarqube:
   image: sonarqube
   ports:
   - "19000:9000"
   - "19092:9092"
   links:
     - sonardb
   environment:
      - SONARQUBE_JDBC_URL=jdbc:postgresql://sonardb:5432/sonar

sonardb:
   image: postgres:latest
   ports:
     - "5433:5432"
   environment:
     - POSTGRES_USER=sonar
     - POSTGRES_PASSWORD=sonar
```
works as expected - SonarQube communicates with PostgreSQL using port **5432** which is published for the all other world as 5433.

**Comment by IAPOLINARIO** [comment]
Uowwww...That's great! I've read about the containers link, but i didn't realize that the communication occurs internally and the port mapping it's just to externally world. This scenario it's a little trick. 

I spent all day trying to connect to Postgres and my last move was make a post here. Thank's a lot!

I made the change in my docker-compose.yml and works perfectly! 

PS - Using 

> depends_on

 works in the same way.
